### PR TITLE
made a server and seeded database

### PR DESCRIPTION
**Pull request by nimaester**

### CODE REVIEW

File: db/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-console */
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'reborn7528',
```
Comment (jgarcia260) [inline db/index.js:7]: look into changing the root password to none

File: db/schema.sql [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE comments (
+  id INT NOT NULL AUTO_INCREMENT,
+  reviewsId INT,
+  comment VARCHAR(255),
+  FOREIGN KEY (reviewsId) REFERENCES reviews(id),
+  PRIMARY KEY(id)
+);
```
Comment (jgarcia260) [inline db/schema.sql:41]: very good variable names

File: db/seed.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+];
+
+let houseName = faker.address.streetName();
+let name = faker.name.findName();
+let url = `https://7528userurl.s3-us-west-1.amazonaws.com/userImage${rdm(1, 110)}.jpg`;
```
Comment (jgarcia260) [inline db/seed.js:25]: GREAT USE OF STRING LITERALS. very nice

File: db/seed.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+const seedReviews = () => {
+  for (let i = 0; i <= 3000; i ++) {
+    let review = faker.lorem.sentences();
+    let postQuery = `INSERT INTO reviews(userId, review, houseId, cleanRating, accRating, commRating, locationRating, checkInRating, valueRating) VALUES(${rdm(1, 200)}, "${review}", ${rdm(1, 100)}, ${rdm(3,5)}, ${rdm(3,5)}, ${rdm(3,5)}, ${rdm(3,5)}, ${rdm(3,5)}, ${rdm(3,5)})`;
+    db.query(postQuery, (err, data) => {
+      if (err) {
+        console.log('error', err);
+      }
+    })
+  }
```
Comment (jgarcia260) [inline db/seed.js:66]: linter?

File: server/index.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+const model = require('../db/model.js');
+const parser = require('body-parser')
+const path = require('path');
+
+app.use('/', express.static(path.join(__dirname, '../client/dist')));
+app.use(parser.urlencoded({extended: false}));
+
+app.listen(port, () => console.log(`Listening At Port ${port}`))
+
+app.get('/reviews', (req, res) => {
```
Comment (jgarcia260) [inline server/index.js:13]: create a routes folder for requests

**Review by jgarcia260 — CHANGES_REQUESTED** [review]
very clean code. create a routes file for incoming requests

File: db/seed.js (hunk 4 above)
Comment (nimaester) [reply db/seed.js:66]: none.

File: server/index.js (hunk 5 above)
Comment (nimaester) [reply server/index.js:13]: changing stuff here

**Review by jgarcia260 — APPROVED** [review]

**Pull request merged**